services/api/src/api/app.py

```diff
@@ -75,7 +75,7 @@ def create_app() -> Starlette:
         Route(
             endpoint_name,
             endpoint=create_endpoint(
```

Comment:
it might be good to pass endpoint_name as a parameter for logging and error messages purposes (see comment below)

Reply:
Done